source/reference/api/start.txt

```diff
@@ -111,16 +111,38 @@ Request Body Parameters
 
        Default value is ``false``.
 
+   * - ``includeNamespaces``
+     - array
+     - Optional
+     - Filters the databases or collections to sync. The filter document
+       syntax is: 
+
+       .. include:: /includes/api/facts/includeNamespaces-syntax.rst
+
+       .. versionadded:: 1.1
+
+       If you configure a filter on a source cluster that has multiple
+       databases, ``mongosync`` only syncs the databases specified in
+       the filter definition. ``mongosync`` will not sync the other
+       existing databases.
+       
+       If you want to modify the filter to add a newly created database,
+       you have to :ref:`restart the filtered sync <c2c-change-filter>`
+       from the beginning.
+
+       For more details, see: :ref:`c2c-filtered-sync`.
+
+       For current limitations, see: :ref:`c2c-filtering-limitations`.
```

Comment:
conform to "Filtered Sync"

Reply:
There's no copy change here. The link pulls in the title from the updated reference below.